.vscode/launch.json

```diff
@@ -3,22 +3,13 @@
     {
       "type": "node",
       "request": "launch",
-      "name": "Spec Test Current File",
-      "cwd": "${workspaceFolder}/stencil-library",
-      "program": "${workspaceFolder}/stencil-library/node_modules/.bin/stencil",
-      "args": ["test", "--spec", "--", "${fileBasename}"],
+      "name": "Debug Jest Tests",
```

Comment:
I think no one was using the old two configurations for local debugging and they were also outdated since the directory restructure.